sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb

```diff
@@ -0,0 +1,955 @@
+{
```

Comment:
<p>I think we can do these install when starting with analysis. Few less steps before getting started.</p><p>Initial training start should not require this install. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
may be just move smdebug install to later part of script where we do analysis.